operations/helm/charts/grafana-agent/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: grafana-agent
 description: "Grafana Agent"
 type: application
-version: 0.1.0
+version: 0.1.1
```

Comment:
Can you bump this up to 0.2.0 instead since it's technically not fixing a bug?

Reply:
Yep, will do; wasn't sure what version to make it when bumping.